k8s-daemonset/k8s/api.yml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: api
+spec:
+  replicas: 3
+  selector:
+    app: api
+  template:
+    metadata:
+      labels:
+        app: api
+    spec:
+      dnsPolicy: ClusterFirst
+      containers:
+      - name: service
+        image: buoyantio/helloworld:test7
```

Comment:
Good question! My preference is to keep all of the configs pinned at the same version.